examples/jsm/loaders/PLYLoader.js

```diff
@@ -407,16 +408,12 @@ class PLYLoader extends Loader {
 
 			}
 
-			if ( buffer.colors.length > 0 ) {
```

Comment:
I think this change introduced a new bug. Now, I am not able to render a point cloud in ply format with color information. Looking at the code, the buffer color is filled but its value is not used anywhere.

Reply:
Yes, it seems this `if` clause should not be deleted. Would you mind filing a PR and revert the deletion?

Reply:
@SindreYang do you mind taking care of this?

Reply:
Never mind! Already fixed it here: #26214